src/ui/views/objectBrowser.ts

```diff
@@ -737,7 +737,9 @@ export function initializeObjectBrowser(context: vscode.ExtensionContext) {
         });
 
         if (changeResult.code === 0) {
+          //pre updating description to avoid old description when multiple updates are performed without refreshing 
           node.description = newText.toUpperCase() !== `*BLANK` ? newText : ``;
+          node.member.text = newText.toUpperCase() !== `*BLANK` ? newText : ``;
```

Comment:
Just assign node.description to node.member.text here, so you don't copy/paste the same logic twice 😉
```suggestion
          node.member.text = node.description;